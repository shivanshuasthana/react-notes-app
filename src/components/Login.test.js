import React from 'react';
import { mount } from 'enzyme';
import Login from './Login';

test('should render Login component correctly', () => {
    const wrapper = mount(
        <Login />
    );
    expect(wrapper).toMatchSnapshot();
});