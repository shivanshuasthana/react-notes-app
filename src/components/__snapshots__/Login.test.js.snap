// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Login component correctly 1`] = `ReactWrapper {}`;
